src/screens/Mentor/Meetup.tsx

```diff
@@ -222,15 +227,10 @@ export default function Meetup({ mentor }) {
                 onClick={async () => {
                   setSelectedDay(index)
                   setSelectedSlot(null)
-                  await setTimeout(
-                    () =>
-                      window.scrollTo({
-                        left: 0,
-                        top: document.body.scrollHeight,
-                        behavior: 'smooth',
-                      }),
-                    200
-                  )
+                  await setTimeout(() => {
```

Comment:
You can't await `setTimeout`. `setTimeout` doesn't return a promise, it returns the id of the timeout, which you can pass to `clearTimeout` to cancel the timeout.

Reply:
I removed it.